Common/Basic components/Filters.cs

```diff
@@ -0,0 +1,52 @@
+﻿using OpenQA.Selenium;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Infrastructure
+{
+    public class Filters : ComponentBase
+    {
+        public List<Button> ColorFilter => parentElement.FindElements(By.CssSelector("#ul_layered_id_attribute_group_3 li")).Select(element => new Button(driver, element)).ToList();
+        public IWebElement PriceRange => parentElement.FindElement(By.CssSelector("#layered_price_range"));
+
+
+        public Filters(IWebDriver driver, IWebElement parentElement) : base(driver, parentElement)
+        {
+
+        }
+
+        public double[] GerPriceRange()
+        {
+            var priceRange = PriceRange.Text;
+            var lowestPrice = new List<char>();
+            var highestPrice = new List<char>();
```

Comment:
Try not to use vars, you can press Control+Enter and ask it to show the explicit type of the field instead of var